app/main.py

```diff
@@ -1,25 +1,71 @@
 from abc import ABC
+from typing import Any
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: Any, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, obj: Any, objtype: Any = None) -> int:
+        return getattr(obj, self.protected_name)
+
+    def __set__(self, obj: Any, value: int) -> None:
+        if value in range(self.min_amount, self.max_amount + 1):
+            setattr(obj, self.protected_name, value)
+        else:
```

Comment:
else: is redundant in this case

Reply:
but if I delete else block or just "else", code doesn't work any more.